policyengine_us/variables/gov/states/sc/tax/income/sc_state_tax_addback.py

```diff
@@ -0,0 +1,50 @@
+from policyengine_us.model_api import *
+
+
+class sc_state_tax_addback(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "South Carolina State Tax addback"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://dor.sc.gov/forms-site/Forms/SC1040_2022.pdf#page=2",
```

Comment:
add one source that shows the legal code (law)

Reply:
Hi @PavelMakarchuk, does [this](https://www.scstatehouse.gov/code/t12c006.php) (SECTION 12-6-50(17)) counts as the legal code? This legal code states that the way SC calculated the itemized deduction is different from the federal's. Other things such as the capped amount used in our code are all followed by federal standards.